sklearn/linear_model/_base.py

```diff
@@ -371,22 +371,26 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
     """
     Ordinary least squares Linear Regression.
 
+    LinearRegression fits a linear model with coefficients w = (w1, ..., wp)
+     to minimize the residual sum of squares between the observed targets in
+     the dataset, and the targets predicted by the linear approximation.
```

Comment:
These two lines need to be indented the same way as the third line above. That's probably why CI is currently failing.

Reply:
Latest commit indented in same line now. Thanks.